src/Components/MovieList/MovieSearch.test.js

```diff
@@ -8,11 +8,11 @@ import {
   waitFor,
   screen,
   cleanup,
+  act,
```

Comment:
We actually want to import `act` from react-testing-lib, not react-dom/test-utils. 
The react-testing-lib version is kind of a wrapper around the react-dom one anyway